pyproject.toml

```diff
@@ -1,2 +1,47 @@
+[build-system]
+requires = ["setuptools>=45", "setuptools_scm[toml]>=7"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "conda-smithy"
+authors = [
+    {name = "Phil Elson", email = "pelson.pub@gmail.com"},
+]
```

Comment:
Wonder if we should move to [`maintainers`]( https://peps.python.org/pep-0621/#authors-maintainers ) with a new email. Admittedly not critical right now

Reply:
Raised as issue ( https://github.com/conda-forge/conda-smithy/issues/1714 )